src/controller/deliveryEmployeeController.ts

```diff
@@ -0,0 +1,47 @@
+import { Application, Request, Response } from "express";
+import { DeliveryEmployee } from "../model/deliveryEmployee";
+
+const deliveryEmployeeService = require('../service/deliveryEmployeeService');
+
+module.exports = function(app: Application) {
```

Comment:
You could make this more typescript and user friendly by doing the following:

Replace module.exports with
```
export const deliveryEmployeeController = (app: Application) {
   ... code here
}
```

In app.ts you can import the function using:
`import {  deliveryEmployeeController } from 'path'`

You can then setup the routes by using
`deliveryEmployeeController(app)`